### PR TITLE
chore: fix typos

### PR DESCRIPTION
**Pull request by ACGNnsj**
🙂I've fixed typos in some files.

### CODE REVIEW

File: cli/tsc/README.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ currently (unfortunately) have a rather manual process for upgrading TypeScript.
...
-1. Copy typescript.js into Deno repo.
-1. Copy d.ts files into dts directory.
-1. Review the copied files, removing and reverting what's necessary
+2. Add Microsoft/TypeScript as a remote and fetch its latest tags
+3. Checkout a new branch based on this tag.
+4. Cherry-pick the custom commit we made in a previous release to the new one.
+5. This commit has a "deno.ts" file in it. Read the instructions in it.
+6. Copy typescript.js into Deno repo.
+7. Copy d.ts files into dts directory.
+8. Review the copied files, removing and reverting what's necessary
```
Comment (dsherret) [inline cli/tsc/README.md:19]: It being all `1`s is a way to have an automatically numbered list https://stackoverflow.com/a/12145270/188246

**Review by dsherret — APPROVED** [review]
LGTM. Thanks!

**Review by dsherret — CHANGES_REQUESTED** [review]
It seems there's some conflicts now. I fixed them, but then I can't push to your repo because the PR was done from your main branch:

```
To https://github.com/ACGNnsj/deno.git
 ! [remote rejected]     ACGNnsj/main -> ACGNnsj/main (permission denied)
error: failed to push some refs to 'https://github.com/ACGNnsj/deno.git'
```

**Comment by ACGNnsj** [comment]
> confli

Thanks for the reminder, and I've solved the conflicts. 🙂

**Comment by dsherret** [comment]
It's failing the lint step. Can you run `deno run -A ./tools/format.js`?

**Review by dsherret — CHANGES_REQUESTED** [review]
It's failing the lint step. Can you run `deno run -A ./tools/format.js`?

**Review by aapoalas — CHANGES_REQUESTED** [review]
@ACGNnsj Unfortunately there're new conflicts to handle, and format presumably still needs to be run. I'd like to get this merged :)

**Pull request closed without merging**